maintainers/maintainer-list.nix

```diff
@@ -3001,6 +3001,12 @@
     githubId = 9959940;
     name = "Andreas Fehn";
   };
+  felixsinger = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add felixsinger``.

Reply:
Already done in commit 32836182e